react/features/calendar-sync/web/microsoftCalendar.js

```diff
@@ -106,7 +107,7 @@ export const microsoftCalendarApi = {
                 // get .value of every element from the array of results,
                 // which is an array of events and flatten it to one array
                 // of events
-                .then(result => [].concat(...result.map(en => en.value)))
+                .then(result => [].concat(...result))
```

Comment:
this was changed because it was moved to another place in the fetch chain? No change in behavior?

Reply:
Yep, in formatCalendarEntry when adding the calendarId to the items.